checkout/src/test/java/com/checkout/validation/validator/CvvValidatorTest.kt

```diff
@@ -117,10 +117,9 @@ internal class CvvValidatorTest {
 
         @JvmStatic
         fun invalidCardSchemeArguments(): Stream<Arguments> = Stream.of(
-            Arguments.of("", CardScheme.UNKNOWN, provideFailure(ValidationError.CVV_INVALID_CARD_SCHEME)),
-            Arguments.of("12", CardScheme.UNKNOWN, provideFailure(ValidationError.CVV_INVALID_CARD_SCHEME)),
-            Arguments.of("123", CardScheme.UNKNOWN, provideFailure(ValidationError.CVV_INVALID_CARD_SCHEME)),
-            Arguments.of("1234", CardScheme.UNKNOWN, provideFailure(ValidationError.CVV_INVALID_CARD_SCHEME)),
+            Arguments.of("12", CardScheme.UNKNOWN, provideFailure(ValidationError.CVV_INCOMPLETE_LENGTH)),
+            Arguments.of("1", CardScheme.UNKNOWN, provideFailure(ValidationError.CVV_INCOMPLETE_LENGTH)),
+            Arguments.of("12345", CardScheme.UNKNOWN, provideFailure(ValidationError.CVV_INVALID_LENGTH)),
```

Comment:
It would be nice to add test cases for  Unknown card scheme with valid CVV options.
Take a look on `validCvvArguments ()`.